### ISSUE TITLE
TypeScript usage

### ISSUE THREAD

**Issue by glensc** [bramstein/url-template#28]
Unclear how to import this module with typescript.

Dependencies added:

```sh
yarn add url-template
yarn add @types/url-template --dev
```

tried:

```typescript
import urltemplate from "url-template";
import { urltemplate } from "url-template";
import { TemplateParser } from "url-template";
import { Template } from "url-template";
```

all say no such export.

**Comment by wpauser-sps** [comment]
import template from 'url-template'; 

worked for me

**Comment by jonkoops** [comment]
Closed in #39
